### PR TITLE
chore: re-enable certain eslint rules globally

### PR DESCRIPTION
**Pull request by priyajeet**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @priyajeet has signed the CLA. Thanks for the pull request!

File: src/components/badge/Badge.js [hunk 1]
```diff
@@ -8,9 +8,9 @@ type Props = {
     /** Child components */
     children: React.Node,
     /** A predefined badge type */
-    type?: 'info' | 'warning' | 'highlight' | 'error' | 'alert' | 'success',
-    /** Adds a class to the component */
     className?: string,
+    /** Adds a class to the component */
```
Comment (JustinHoldstock) [inline src/components/badge/Badge.js]: I think you moved the wrong comment, here

Comment (priyajeet) [reply src/components/badge/Badge.js]: Yeah I think eslint auto-fix ordering misses comments

**Pull request merged**